### ISSUE TITLE
The `-p` option does not have the alias function

### ISSUE THREAD

**Issue by witt-bit** [tldr-pages/tldr#15382]
`tldr -p linux awk` output : 
```shell
$ tldr -p linux awk                                                                                                  ✔ 
`awk` documentation is not available.
If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr
```

`tldr awk` output : 
```shell
$ tldr awk                                                                                                         1 ✘ 

  awk

  一种用于文件处理的通用编程语言。
  更多信息：https://github.com/onetrueawk/awk.

  - 以空格为分隔符，打印文件每行第五列（也称作字段）：
    awk '{print $5}' 文件名

  - 以空格为分隔符，打印文件包含“foo” 的所有行的第二列：
    awk '/foo/ {print $2}' 文件名

  - 以逗号而不是空格作为分隔符，打印文件每行的最后一列：
    awk -F ',' '{print $NF}' 文件名

  - 计算文件的第一列数值之和并打印：
    awk '{s+=$1} END {print s}' 文件名

  - 从第一行开始，每三行打印一行：
    awk 'NR%3==1' 文件名

  - 根据条件不同，打印不同内容：
    awk '{if ($1 == "foo") print "Exact match foo"; else if ($1 ~ "bar") print "Partial match bar"; else print "Baz"}' 文件名

  - 打印第 10 列等于指定值的所有行：
    awk '($10 == 指定值)'

  - 打印第 10 列介于最小值和最大值之间的所有行：
    awk '($10 >= 最小值 && $10 <= 最大值)'
```

The `awk` command is located in `common/awk.md` and should work on any platform (`linux`, `macos`, etc.), at least no matter what the `-p` option is, you should get the documented results.

-----------------------------------------------------------------------------------------------

The reason why I have this problem is that I execute the command `tldr gawk` and the output is

```shell
$ tldr gawk                                                                                                          ✔ 

  gawk

  这是 `-p linux awk` 命令的一个别名。

  - 原命令的文档在：
    tldr -p linux awk

Found 2 pages with the same name under the platforms: osx, osx.
```

Leads me to execute `tldr -p linux awk`,After I executed it, the following prompts
```shell
$ tldr -p linux awk                                                                                                  ✔ 
`awk` documentation is not available.
If you want to contribute it, feel free to send a pull request to: https://github.com/tldr-pages/tldr
```

I think this is ill-designed!

**Comment by gutjuri** [comment]
Which tldr client are you using (aka how did you install tldr?). Sounds like a bug in the client. I interpret the client specification (https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#page-resolution) as requiring that a page should be displayed when it exists for any platform, even when the desired platform is specified through the `-p` parameter.

**Comment by witt-bit** [comment]
OS: `Manjaro 24.2.1 Yonada`, and install `tldr` using the command `sudo pacman -S tldr`, I understand your description, but I think this may be a more usable optimization point.

**Comment by acuteenvy** [comment]
It is a bug in the client, but the `gawk` page is wrong too. There is no page for `awk` in the `linux` directory.
